lls/main.c

```diff
@@ -224,13 +253,46 @@ lls_scan(__attribute__((unused)) struct rte_timer *timer, void *arg)
 		lls_cache_scan(lls_conf, &lls_conf->nd_cache);
 }
 
+/* Send gratuitous ARP or ND announcement to fulfill LACP TX requirement. */
 static void
+lls_lacp_announce(__attribute__((unused)) struct rte_timer *timer, void *arg)
+{
+	struct lls_config *lls_conf = (struct lls_config *)arg;
+	struct net_config *net_conf = lls_conf->net;
+	if (lacp_enabled(net_conf, &net_conf->front)) {
+		if (iface_arp_enabled(net_conf, &net_conf->front))
+			xmit_arp_req(&net_conf->front,
+				(const uint8_t *)&net_conf->front.ip4_addr,
+				NULL, lls_conf->tx_queue_front);
+		else if (likely(iface_nd_enabled(net_conf, &net_conf->front)))
```

Comment:
This is the first occasion we want to test IPv6 (i.e. ND) before IPv4 (i.e. ARP): ARP broadcast gratuitous packets whereas ND multicast gratuitous packets. That is, ND gratuitous packets put less load on the network fabric. Could invert the test here and below?